runtime/src/lib.rs

```diff
@@ -177,7 +177,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("polkadex-parachain"),
 	impl_name: create_runtime_str!("polkadex-parachain"),
 	authoring_version: 1,
-	spec_version: 3,
+	spec_version: 6,
```

Comment:
should we jump 3 versions for release? (probably incremented for testnet)

Reply:
Current parachain version is same